README-CONAN.md

```diff
@@ -1,108 +1,193 @@
-# How to use conan
+# How to use conan to build Exiv2 and dependencies
 
-Conan is a portable package manager for C/C++ libraries. It can be used for bringing all the dependencies needed to
+Conan is a portable package manager for C/C++ libraries. It can be used to bring all the dependencies needed to
 build Exiv2 into local directories, without needing to install system packages.
 
 This document provides a step-by-step guide to show you the basic usage of conan. For more details about the tool,
-please visit their [documentation website](http://docs.conan.io/en/latest/).
+please visit the [Conan documentation website](http://docs.conan.io/en/latest/).
+
+Although we provide step-by-step instructions to enable you to build Exiv2 with conan, we strongly recommend that you
+read conan's documentation to understand the main concepts: [Getting started with Conan](http://docs.conan.io/en/latest/getting_started.html)
+
+To build Exiv2 with conan, you will also need to install CMake.  For more information see [README-CMAKE](README-CMAKE)
+
+<name id="TOC"></a>
+----
+### T A B L E &nbsp;&nbsp;&nbsp; O&nbsp;&nbsp;F &nbsp;&nbsp;&nbsp; C O N T E N T S
```

Comment:
Are the `&nbsp;` intentional or just quirks added by your editor?

Reply:
The &nbsp; is quite deliberate to make the formatting look better.

My suspicion is that MarkDown is a language that passes almost everything to the generated HTML.  If MarkDown doesn't recognise something, it copies it to generated HTML.  In README.md, I wanted to use multiple columns in a table.  When I googled into this, the syntax is `<td colspan="2">.....</td>` which is pure HTML.

So, I think MarkDown is a preprocessor that makes it easy to define HTML tags such as `<h1>` using `#`, however for more control of the HTML, you can simply insert HTML and it'll be passed unmodified.